acasclient/acasclient.py

```diff
@@ -1383,3 +1385,98 @@ def get_blob_data_by_value_id(self, valueId):
                                 .format(self.url, valueId))
         resp.raise_for_status()
         return resp.content
+    
+    def get_authors(self):
+        """
+        Get all authors
+
+        Returns:
+            a list of dict objects representing the authors
+        """
+        resp = self.session.get("{}/api/authors".format(self.url))
+        resp.raise_for_status()
+        return resp.json()
+    
+    def get_author_by_username(self, username):
+        """
+        Get author by username
+            
+        Args:
+            username (str): The username of the author to fetch
+
+        Returns:
+            a dict object representing the author
+        """
+        resp = self.session.get("{}/api/authorByUsername/{}".format(self.url, username))
+        resp.raise_for_status()
+        return resp.json()
+    
+    def create_author(self, author):
+        """
+        Create an author
+
+        Args:
+            author (dict): A dict object representing the author to create
+
+        Returns:
+            a dict object representing the new author
+        """
+        def hash_password(password):
+            """ Calculate the base64-encoded sha1 hash of the password for ACAS built-in authentication """
+            hasher = hashlib.sha1()
+            hasher.update(password.encode('utf-8'))
+            return base64.b64encode(hasher.digest()).decode('utf-8')
+        if 'password' in author:
```

Comment:
I appreciate this comment and the thought it provokes. Right now I don't think it's worth defining these as constants, since I don't see a lot of repetition of strings _across_ different functions. If we later migrate more of this towards object-oriented classes, that will be a good time to define these constants in a single place.